tests/bud.bats

```diff
@@ -290,7 +290,8 @@ _EOF
          "With explicit --platform, buildah should warn about pulling difference in platform"
   assert "$output" =~ "TARGETOS=linux" " --platform TARGETOS set correctly"
   assert "$output" =~ "TARGETARCH=amd64" " --platform TARGETARCH set correctly"
-  assert "$output" =~ "TARGETVARIANT=" " --platform TARGETVARIANT set correctly"
+  # By default, BUILDVARIANT/TARGETVARIANT should be empty.
+  assert "$output" =~ "TARGETVARIANT=\s" " --platform TARGETVARIANT set correctly"
```

Comment:
Is the `\s` a perl-ism?  I'm surprised the shell is able to match it in the way this test expects.

Reply:
`=~` is a regex operator. So I'm trying to match the end of the line with `\s`.

Reply:
I understand that, but `\s` isn't something I can find described in `regex(7)`.  It pops up in several man pages related to Perl, though.

Reply:
Interesting, I never thought of that. Is that wrong? Should I use `[[:cntrl:]]` instead?